.github/workflows/checks.yaml

```diff
@@ -3,10 +3,10 @@ name: lint and test
 on:
   push:
     branches:
-      - '**'
+      - "**"
```

Comment:
I suppose, my automatic formatter on ctrl+s worked here. Generally, do we have some yaml code-style rules established?

Reply:
We don't have a code-style rules.
In some repositories we're applying a yaml linter action which isn't the best (because it doesn't have --fix option)
but AFAIR this action also complains about usage of `"` instead of `'` so this is the rule you should set in your formatter.

Reply:
So, we should left this change, I suppose.